### PR TITLE
Popup design DOM

### PR DESCRIPTION
**Pull request by Kajal19-del**
The goal of this project is to build a straightforward, responsive, single-page portfolio that will hold all of the past personal projects. You must configure the HTML-CSS linter for this project in accordance with the instructions provided in the HTML-CSS setup instructions and build the website with mobile users in mind.

# In this project I did the following activities:
- I add JS linters files to the project
- We Implement DOM to create this popup design
- I use javascript to create HTML section structure
- I get help from Figma to complete this design

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -31,4 +31,158 @@ navLinks.forEach((link) => {
...
+      </div>
+      <p class="section-text">${item2.para1}</p>
+      <ul class="section-skills">
+        <li class="skills">${item2.tech[0]}</li>
+        <li class="skills">${item2.tech[1]}</li>
+        <li class="skills">${item2.tech[2]}</li>
+      </ul>
+      <a href="javascript:void(0)" class="btn-text" name="${item2.id}">See Project</a>
+    </div>
+  </div>
```
Comment (omarsalem7) [inline app.js:123]: - [ ] Kindly adjust the design for the image for the project to match the Figma design.

| Your design  | Figma design|
| ------------- | ------------- |
| image | image |

File: app.js [hunk 2]
```diff
@@ -31,4 +31,158 @@ navLinks.forEach((link) => {
...
+                  <a href="${card[0].live}" class="btn-text" name="1">See live <img class="imgButton" src="./images/live.png" alt="live" width="24"></a>
+                  <a href="${card[0].source}" class="btn-text" name="1">See Source <img class="imgButton" src="./images/IconGitHub.png" alt="IconGitHub" width="24"></a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    `;
+    popUp.classList.toggle('hide');
```
Comment (omarsalem7) [inline app.js]: - [ ] Kindly make sure when you click on the see project button the corresponding popup modal contains info about the opened project with details about it not the same one for every modal.

- [ ] Also consider adjusting the size of the image in the modal to match the one in the Figma design, kindly take a look at the following image for Modal in the Figma design 
>image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Kajal19-del 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (Kajal19-del) [reply app.js:123]: Hello @omarsalem7, can you please clarify the changes please. Is it the image size you want me to change?

**Review by mikethreels — APPROVED** [review]
Hi Kajal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**